com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md

```diff
@@ -1,230 +0,0 @@
-# Summary
```

Comment:
Deleted the doc since most of the information here doesn't apply anymore.
Only CountingGridSensor is in extension now and I don't think that worth a whole doc.